oat-sa/tao-item-runner-qti-fe
# allow interaction

Ticket: https://oat-sa.atlassian.net/browse/MS-532

### What has changed?

- Functionality to allow interaction in specific PCIs

### how to test?

- Create a test with text reader PCI and one more PCI (don't forget to add manual scoring flags)
- Publish the test and create a scoring project
- Give the test as a TT
- Login as a scorer on MS and try to interact with both the PCIs
- You should be able to interact with text reader but not wiht any other PCIs

### Related

https://github.com/oat-sa/extension-tao-itemqti/pull/1593
